src/exif.c

```diff
@@ -142,6 +142,48 @@ avifResult avifImageExtractExifOrientationToIrotImir(avifImage * image)
     return AVIF_RESULT_OK;
 }
 
+uint8_t avifImageIrotImirToExifOrientation(const avifImage * image)
```

Comment:
Thank you for the snippet. It would require checking `axis` and `angle` values before using them as array indices though.

Keeping the current version as it is already done.